### ISSUE TITLE
Instructions unclear

### ISSUE THREAD

**Issue by JasonCard** [MicrosoftDocs/visualstudio-docs#5882]
Moving issue from [https://github.com/MicrosoftDocs/visualstudio-docs.de-de/issues/95](https://github.com/MicrosoftDocs/visualstudio-docs.de-de/issues/95) as feedback for English content.

castilma commented 10 hours ago
Die Seite enthält die info Der Knoten Allgemein ist auch sichtbar, wenn die Option Alle Einstellungen anzeigen deaktiviert ist. Wenn Sie alle verfügbaren Optionen anzeigen möchten, klicken Sie auf Alle Einstellungen anzeigen.
Aber es ist nicht klar, wo 'Alle Einstellungen anzeigen' angeklickt werden soll.

ghogen commented 9 hours ago
The user is saying that "Show All Settings" checkbox cannot be found in the Tools, Options dialog box. @TerryGLee - it seems that maybe there used to be a checkbox on the Tools Options dialog called "Show All Settings" but maybe it's not there anymore. We still have docs that reference it that might need to be updated. Here's another example, https://docs.microsoft.com/en-us/visualstudio/ide/reference/options-dialog-box-visual-studio

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2ad068e6-6b02-6800-e9c5-601fd7b765b7
* Version Independent ID: a7034303-7838-54a8-4c3b-cc3695e65d1d
* Content: [Manage build configurations with Visual Basic developer settings](https://docs.microsoft.com/en-us/visualstudio/ide/how-to-manage-build-configurations-with-visual-basic-developer-settings-applied?view=vs-2019)
* Content Source: [docs/ide/how-to-manage-build-configurations-with-visual-basic-developer-settings-applied.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/ide/how-to-manage-build-configurations-with-visual-basic-developer-settings-applied.md)
* Product: **visual-studio-windows**
* Technology: **vs-ide-compile**
* GitHub Login: @ghogen
* Microsoft Alias: **ghogen**

**Comment by JasonCard** [comment]
@castilma @ghogen @TerryGLee FYI, moved issue from MicrosoftDocs/visualstudio-docs.de-de#95 as feedback for English content.

**Comment by TerryGLee** [comment]
@JasonCard @ghogen I have updated all three pages in our content that refer to this deprecated feature. The updated pages should be live tomorrow, shortly after 10:00am Pacific time on 9/24/2020. Thanks for the heads-up!

@castilma Wir freuen uns, dass Sie darauf hingewiesen haben; Wir haben unsere englischen Inhalte aktualisiert, die bald in alle anderen unterstützten Sprachen lokalisiert werden sollten. Wir danken Ihnen, dass Sie uns geholfen haben, die Dokumentation zu verbessern!

(Translation: "We appreciate you pointing this out; we have updated our English content, which should soon be localized into all other supported languages. We thank you for helping us improve the documentation!")
